packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/CommunityTypeStep/helpers.ts

```diff
@@ -52,7 +52,7 @@ export const communityTypeOptions = [
     chainBase: ChainBase.Solana,
     title: 'Solana',
     isRecommended: false,
-    isHidden: true,
+    isHidden: false,
```

Comment:
in second step (form), there is a logic that sets default chain from the dropdown depending on the type of the community selected in step 1. Can't test it as I am not able to connect phantom wallet, but just wanna make sure that it works and that solana community can be created successfully

Reply:
Yes the full create community flow works as expected for me. You may need to try to update/re-install phantom as it worked out of the box on my local set up many times as well as production

Reply:
okay there was some problem with my wallet. I reinstalled it and it worked fine. Was able to test the flow, created community sucessfully